pyasn/mrtx.py

```diff
@@ -115,6 +115,15 @@ def dump_prefixes_to_text_file(ipasn_data, out_text_file_name, orig_mrt_name, de
         fw = open(out_text_file_name, 'wt')
     else:
         fw = open(out_text_file_name, 'wt', encoding='ASCII')
+
+    fw.write(dump_prefixes_to_text(ipasn_data, orig_mrt_name, debug_write_sets=debug_write_sets))
+
+    fw.close()
+
+
+def dump_prefixes_to_text(ipasn_data, orig_mrt_name, debug_write_sets=False):

```

Comment:
This change should break a test, or write a test that breaks without this change.